### PR TITLE
Add documentation about trusting self-signed certs on iOS

### PR DESCRIPTION
**Pull request by hobinjk-ptc**

### CODE REVIEW

**Comment by ptc-rdeleeuw** [comment]
this only concerns the documentation but there is no mention of how to start the python webserver mentioned in the documentation. I know this is easy to do, but we might want to add a shell script to the scripts folder or add the commandline starting the python server in the documentation it self

File: docs/ios-cert-install.md
Comment (ptc-rdeleeuw) [inline docs/ios-cert-install.md:1]: this only concerns the documentation but there is no mention of how to start the python webserver mentioned in the documentation. I know this is easy to do, but we might want to add a shell script to the scripts folder or add the commandline starting the python server in the documentation it self

Comment (hobinjk-ptc) [reply docs/ios-cert-install.md:1]: Oh sorry, you can see https://github.com/ptcrealitylab/vuforia-spatial-edge-server/pull/1009/files#diff-95c07260d1a629a46230701e387127ad04f277913dc2b8cb78e955a460edd17cR14 is where I meant to put the snippet

Comment (hobinjk-ptc) [reply docs/ios-cert-install.md:1]: (added)

**Review by ptc-rdeleeuw — APPROVED** [review]

**Pull request merged**